### PR TITLE
doc: Enhance HTTP-Pull Documentation

### PR DESCRIPTION
**Pull request by efiege**
_What issues does this PR close?_
https://github.com/sovity/edc-extensions/issues/728

```[tasklist]
### Checklist
- [x] The PR title is short and expressive.
- [x] I have updated the CHANGELOG.md. See [changelog_update.md](https://github.com/sovity/edc-extensions/tree/main/docs/dev/changelog_updates.md) for more information.
- [x] I have updated the Deployment Migration Notes Section in the CHANGELOG.md for any configuration / external API changes.
- [x] I have performed a **self-review**
```

### CODE REVIEW

File: docs/getting-started/documentation/pull-data-transfer.md [hunk 1]
```diff
@@ -34,11 +34,15 @@ When initiating the transfer, select "Custom Transfer Process JSON", and provide
 ```json
 {
   "@type": "https://w3id.org/edc/v0.0.1/ns/TransferRequest",
+  "https://w3id.org/edc/v0.0.1/ns/assetId": "{{ASSET_ID}}",
+  "https://w3id.org/edc/v0.0.1/ns/contractId": "{{AGREEMENT_ID}}",
+  "https://w3id.org/edc/v0.0.1/ns/connectorAddress": "{{COUNTER_PARTY_DSP_ENDPOINT}}",
+  "https://w3id.org/edc/v0.0.1/ns/connectorId": "{{COUNTER_PARTY_ID}}",
```
Comment (richardtreier) [inline docs/getting-started/documentation/pull-data-transfer.md:40]: The UI API Wrapper is called with a contract ID. I believe it is extracting this metadata on the backend side and adding it to the Eclipse EDC JSON-LD before the service call.

File: docs/getting-started/documentation/pull-data-transfer.md [hunk 2]
```diff
@@ -34,11 +34,15 @@ When initiating the transfer, select "Custom Transfer Process JSON", and provide
 ```json
 {
   "@type": "https://w3id.org/edc/v0.0.1/ns/TransferRequest",
+  "https://w3id.org/edc/v0.0.1/ns/assetId": "{{ASSET_ID}}",
```
Comment (richardtreier) [inline docs/getting-started/documentation/pull-data-transfer.md:37]: documentation changes fixes are Patch changes for the CHANGELOG, we can add an entry for that and prefix it with "Docs: "

**Review by richardtreier — CHANGES_REQUESTED** [review]

File: docs/getting-started/documentation/pull-data-transfer.md (hunk 1 above)
Comment (efiege) [reply docs/getting-started/documentation/pull-data-transfer.md]: Thanks for this information. I was also able to start the http-pull without the `contractId`.

**Review by richardtreier — APPROVED** [review]
LGTM

**Pull request merged**